R/postprocess.R

```diff
@@ -1,3 +1,137 @@
+#' Extract samples of the delay distribution parameters
+#'
+#' @param fit A model fit with `epidist::epidist`
+#' @inheritParams brms::prepare_predictions
+#' @param ... Additional arguments passed to `brms::prepare_predictions`
+#' @family postprocess
+#' @autoglobal
+#' @export
+predict_delay_parameters <- function(fit, newdata = NULL, ...) {
+  args <- list(...)
+  if ("newdata" %in% names(args)) {
+    newdata <- brms:::validate_newdata(newdata)
```

Comment:
now that newdata isn't in ... I think it isn't being passed to `brms::prepare_predictions` if this is the case and it wasn't caught by testing this suggests a test is missing we might want to have.

Reply:
Fixed